TheMovieDb/Controllers/MovieCellController.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  MovieCell.swift
+//  TheMovieDb
+//
+//  Created by Misael Chávez on 01/11/21.
+//
+
+import UIKit
+import Kingfisher
+
+class MovieCellController: UICollectionViewCell {
```

Comment:
Nitpick: A Collection View Cell inherits from `UIView`, so it is a view and not a ViewController, so a more adequate name would be `MovieViewCell` or something like that.

Reply:
You're right, will change it